### PR TITLE
Merge "ui" package into "amo"

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-frontend/issues/9982

### CODE REVIEW

File: src/amo/components/App/styles.scss [hunk 1]
```diff
@@ -1,4 +1,3 @@
-@import '~amo/css/inc/lib';
```
Comment (willdurand) [inline src/amo/components/App/styles.scss:1]: Merged with `styles.scss`

File: src/ui/css/vars.scss [hunk 2]
```diff
@@ -1,65 +0,0 @@
...
-// Category Colors
-// There are currently 12 colors and this number must be kept in sync with
-// `src/amo/components/Categories/index.js`.
-$category-colors: $teal-70, $red-60, $magenta-60, $orange-60, $blue-50,
-  $green-70, $magenta-70, $red-50, $purple-40, $orange-70, $blue-60, $yellow-70;
-
-// Fonts
-$font-size-s: 12px;
-$font-size-m-smaller: 14px;
-$font-size-m: 18px;
```
Comment (willdurand) [inline src/ui/css/vars.scss:47]: This is not consistent with what we have in the `amo` css... Other values are correct so now former "UI" components will use the `size-m` of the `amo` css (which is 16px).

**Review by bobsilverberg — APPROVED** [review]
Looks good, thanks @willdurand.

r+wc

File: src/amo/css/inc/vars.scss [hunk 3]
```diff
@@ -40,16 +41,24 @@ $font-size-hero-body: 21px;
...
 $transition-medium: $transition-short * 3;
 $transition-long: $transition-short * 7;
+$easing-curve: cubic-bezier(0.07, 0.95, 0, 1);
 
 // Border-Radius Variables
 $border-radius-default: 9px;
 $border-radius-xs: 2px;
 $border-radius-s: 4px;
 $border-radius-m: 8px;
+$ui-border-radius-l: 12px;
```
[the remark this answers is not in the document]
Comment (willdurand) [reply src/amo/css/inc/vars.scss:61]: ah yep, good call

**Pull request merged**